try:
    result = 45/5
except:
    print("Error happen")
finally:
    print("finally here")
print("Done")